src/features/profile/Profile.jsx

```diff
@@ -0,0 +1,33 @@
+import { DisplayProfile } from "./components/DisplayProfile";
+import { useSelector } from "react-redux";
+import { useParams } from "react-router-dom";
+import { DisplayPost } from "features";
+import { Center, Spinner } from '@chakra-ui/react';
+
+export function Profile(){
+    const {allUsers} = useSelector(store=>store.user);
+    const {posts, isLoading} = useSelector(store=>store.post);
+    const params = useParams();
```

Comment:
suggestion: You can directly destructure params here and use them wherever required.

Reply:
Thanks for the suggestion